script.js

```diff
@@ -0,0 +1,12 @@
+const mobileMenu = document.querySelector('.icon');
+const menuHeader = document.querySelector('.header');
+const menu = document.querySelector('.menu');
+const menuLink = document.querySelectorAll('.link');
+const toggleMenu = () => {
+  menuHeader.classList.toggle('active');
```

Comment:
- [ ] Great job adding the closed menu button as requested by the previous reviewer 👏🏻 but, it would be better if the closed menu button follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). Let's change the closed menu button In order to match the Figma design. You can download the icon image from the template Figma design to make sure you are using the same icon as the original design🎨 👨🏻‍🎨 


- [ ] Kindly remode the extra padding or margin-left of the closed menu button as shown on the screenshot bellow 👍🏻  

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![18](https://user-images.githubusercontent.com/79058364/197126597-0051ebc4-b7ba-441d-a4ca-bafb55807f29.png) | ![18](https://user-images.githubusercontent.com/79058364/197124177-41cb54ca-a762-472a-9838-71b55ff8fe09.png) |